caluma/form/models.py

```diff
@@ -220,6 +233,7 @@ class Meta:
         # a question may only be answerd once per document
         unique_together = ("document", "question")
         indexes = [models.Index(fields=["value"]), models.Index(fields=["date"])]
+        indexes = [GinIndex(fields=["meta", "value"])]
```

Comment:
there are two indexes definitions - there should only be one for this to work properly.

Reply:
oh dear, that was a bad one. would be interesting to get linter warning for that.